internal/commands/diff.go

```diff
@@ -390,6 +395,7 @@ func newDiffCommand(cp configProvider) *cobra.Command {
 	cmd.Flags().StringArrayVar(&config.di.annotationNames, "ignore-annotation", nil, "remove specific annotation from objects before diff")
 	cmd.Flags().BoolVar(&config.di.allLabels, "ignore-all-labels", false, "remove all labels from objects before diff")
 	cmd.Flags().StringArrayVar(&config.di.labelNames, "ignore-label", nil, "remove specific label from objects before diff")
+	cmd.Flags().BoolVar(&config.exitNonZero, "error-exit", true, "exit with non-zero status code when diffs present")
```

Comment:
For some cases like jobs with generated names, this might exit non zero all the time if set.

Reply:
you mean the default behavior and not when explicitly setting the flag, right?